### DISCUSSION TITLE
Webpack 5 BREAKING CHANGE

### DISCUSSION THREAD

**Discussion by sapirel** [webpack/webpack#16657] [Q&A]
I upgraded my react node application  to webpack 5
and I got the following error:

polyfills.122cb91806b0b6ee.js:2 [webpack-dev-server] ERROR in ../../node_modules/@nestjs/common/cache/cache.providers.js 17:108-132
Module not found: Error: Can't resolve 'cache-manager' in '/Users/sapirelisha/Documents/falcon-lab-automation/node_modules/@nestjs/common/cache'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "stream": require.resolve("stream-browserify") }'
	- install 'stream-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "stream": false }

ERROR in ../../node_modules/@nestjs/common/file-stream/streamable-file.js 5:15-30
Module not found: Error: Can't resolve 'util' in '/Users/sapirelisha/Documents/falcon-lab-automation/node_modules/@nestjs/common/file-stream'

polyfills.122cb91806b0b6ee.js:2 [webpack-dev-server] ERROR in ../../node_modules/@nestjs/core/nest-application.js 9:13-26
Module not found: Error: Can't resolve 'os' in '/Users/sapirelisha/Documents/falcon-lab-automation/node_modules/@nestjs/core'

and additional 82 errors like that - could not resolve module.

From google I understand that I should edit webconfig file and  put the module in fallback value but it didn't work .

Any help will be highly appreciated.

**Comment by ankurk91** [comment]
https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5


Google first before creating an issue

**Comment by LoveMuZiLi** [comment]
![image](https://user-images.githubusercontent.com/49928768/213075834-e262b233-75b5-44f3-a821-1280f6b805c3.png)
I have the same problem

**Comment by LoveMuZiLi** [comment]
> https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5
> 
> Google first before creating an issue



> https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5
> 
> Google first before creating an issue

I didn't solve my problem according to this blog

**Comment by LoveMuZiLi** [comment]
![image](https://user-images.githubusercontent.com/49928768/213076095-9a9a6215-e393-4367-a752-9cde64d91306.png)
I configured my code according to the error prompt, but I did not solve my problem

**Reply by alexander-akait** [reply]
What is the problem? Please use the issue template in future

**Comment by nandu901nandu** [comment]
I am also facing the same issue, can you pls check this @alexander-akait @LoveMuZiLi
